boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1251,6 +1251,29 @@ LoadAndBootWindowsCommon(
     /* "Stop all motors", change videomode */
     MachPrepareForReactOS();
 
+    /* Show the "debug mode" notice if needed */
+    BOOLEAN DebugMode = !NtLdrGetOption(BootOptions, "NODEBUG") && /* NODEBUG is absent, priority over DEBUG option */
+                        !NtLdrGetOption(BootOptions, "CRASHDEBUG") && /* CRASHDEBUG is abset, no wait during the boot */
```

Comment:
In `CRASHDEBUG` case, should you warn like "remember to attach a debugger in case it is needed later"?